### PR TITLE
Fix CSP issues in editors QUnit tests

### PR DESCRIPTION
**Pull request by timbset**

### CODE REVIEW

File: testing/tests/DevExpress.ui.widgets.editors/slider.tests.js [hunk 1]
```diff
@@ -280,14 +279,16 @@ module('render', moduleOptions, () => {
             width: 500
         });
 
+        // the width of the right and left margin must be 0 (T927984)
+        $element.find('.' + SLIDER_BAR_CLASS)[0].style = 'margin: 14px 0px;';
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.editors/slider.tests.js]:
```suggestion
        $element.find(`.{SLIDER_BAR_CLASS}`)[0].style = 'margin: 14px 0px;';
```

File: testing/tests/DevExpress.ui.widgets.editors/slider.tests.js [hunk 2]
```diff
@@ -246,14 +247,15 @@ module('render', moduleOptions, () => {
...
-        // the width of the right and left margin must be 0 (T927984)
-        const styles = $('<style>.dx-slider-bar{margin: 14px 0px;}</style>').appendTo($('#qunit-fixture'));
         const $element = $('#slider').dxSlider({
             max: 500,
             min: 0,
             value: 0
         }).css('width', 500);
 
+        // the width of the right and left margin must be 0 (T927984)
+        $element.find('.' + SLIDER_BAR_CLASS)[0].style = 'margin: 14px 0px;';
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.editors/slider.tests.js]:
```suggestion
        $element.find(`.{SLIDER_BAR_CLASS}`)[0].style = 'margin: 14px 0px;';
```

File: testing/tests/DevExpress.ui.widgets.editors/validatorIntegration.tests.js [hunk 3]
```diff
@@ -150,8 +150,14 @@ QUnit.module('Regression', {
         const validator = this.fixture.createTextBoxWithValidator({ validationRules: [{ type: 'required' }] });
         const topDiff = 22;
 
-        $('<div style=\'height: 100px\'/>').insertAfter(
-            this.fixture.$element.wrap('<div id=\'bingo\' style=\'overflow-y: scroll; height: 100px\' />')
+        const $element = $('<div />');
+        $element[0].style = 'height: 100px;';
+
+        const $bingo = $('<div id=\'bingo\' />');
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.editors/validatorIntegration.tests.js:156]:
```suggestion
        const $bingo = $('<div id="bingo" />');
```

**Review by ksercs — DISMISSED** [review]

File: testing/tests/DevExpress.ui.widgets.editors/autocomplete.markup.tests.js [hunk 4]
```diff
@@ -7,10 +7,11 @@ QUnit.testStart(function() {
     const markup =
         '<div id="qunit-fixture" class="dx-viewport">\
             <div id="widget"></div>\
-            <div id="widthRootStyle" style="width: 300px;"></div>\
+            <div id="widthRootStyle"></div>\
         </div>';
 
     $('#qunit-fixture').html(markup);
+    document.getElementById('widthRootStyle').style = 'width: 300px;';
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.editors/autocomplete.markup.tests.js]: to support shadowDOM
```suggestion
    $('widthRootStyle').get(0).style = 'width: 300px;';
```

File: testing/tests/DevExpress.ui.widgets.editors/selectBox.tests.js [hunk 5]
```diff
@@ -44,13 +44,22 @@ QUnit.testStart(() => {
...
                 <div id="selectBoxWithoutScroll"></div>\
             </div>\
-            <div id="test-container" style="overflow-hidden"></div>\
+            <div id="test-container"></div>\
         </div>';
 
     $('#qunit-fixture').html(markup);
+
+    $('#selectBoxWithoutScrollWrapper')
+        .css('position', 'fixed')
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.editors/selectBox.tests.js]:
```javascript
        .css({
            ...
        })
```

File: testing/tests/DevExpress.ui.widgets.editors/autocomplete.tests.js [hunk 6]
```diff
@@ -40,7 +40,8 @@ QUnit.testStart(() => {
         </div>\
     </div>';
 
-    $('#qunit-fixture').html(markup);
+    $('#qunit-fixture').append(markup);
```
Comment (ksercs) [inline testing/tests/DevExpress.ui.widgets.editors/autocomplete.tests.js]: why do we need this change?

**Review by ksercs — APPROVED** [review]

**Review by EugeniyKiyashko — APPROVED** [review]

**Pull request merged**